bosch_locator_bridge/src/locator_bridge_node.cpp

```diff
@@ -74,13 +74,16 @@ void LocatorBridgeNode::init()
   std::string host;
   nh_.getParam("locator_host", host);
 
+  int port;
```

Comment:
The port should be initialized with "8080"

Reply:
The bridge does not support TLS-communication with the Locator docker container. The port should only be changed if the mapped ports of the Locator docker container are shifted.